### PR TITLE
Update test.md

### PR DESCRIPTION
**Pull request by nWODT-Cobalt**
Corrigé quelques coquilles, j'espère ne pas avoir altéré le sens.

### CODE REVIEW

File: dailyperf/tests/icon_border/test.md [hunk 1]
```diff
@@ -13,28 +13,27 @@ _Between border and icon, witch one impact the most the performance?_
...
+ 2. A border color
 
 ## The answer
 
-Between the two choice we have a little performance difference. Border is faster than icon of 20ms in average.
+There's only a little performance difference between the two options. Border is faster than icon by 20ms in average.
 Following the [RAIL performance model](https://developers.google.com/web/tools/profile-performance/evaluate-performance/rail#animation-render-frames-every-16ms) we have a budget of 16 ms for animation.
-In our case, if we have to dynamically generate these events during an animation or an user input (scroll for example), our budget for a fluid animation (16ms) is already burn
-when using icons.
+In our case, dynamically generate these events during an animation or a user input (e.g. scroll) would already use up our all the budget for a fluid animation (16ms).
```
Comment (Tuizi) [inline dailyperf/tests/icon_border/test.md:27]: _"use up our all the budget for a fluid animation"_ 

"use up all our budget for a fluide animation" no?

File: dailyperf/tests/icon_border/test.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ![Border VS Icons](img.png)
 
-_Between border and icon, witch one impact the most the performance?_
+*Between a border and an icon, which one impacts performance the most?*
 
-**tl;dr** No significant impact on performance for both. But border is better than using icon.
+**tl;dr** No significant impact on performance for any. But a border is better than an icon.
```
Comment (Tuizi) [inline dailyperf/tests/icon_border/test.md:7]: _But using border is faster than using icon_
What do you think?

Comment (nWODT-Cobalt) [inline dailyperf/tests/icon_border/test.md:7]: Yeah much better!

File: dailyperf/tests/icon_border/test.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 ![Border VS Icons](img.png)
 
-_Between border and icon, witch one impact the most the performance?_
+*Between a border and an icon, which one impacts performance the most?*
```
Comment (Tuizi) [inline dailyperf/tests/icon_border/test.md:5]: When you have to choose between 2 visuals: border color or icon, which one impacts performance the most?

File: dailyperf/tests/icon_border/test.md [hunk 4]
```diff
@@ -13,28 +13,27 @@ _Between border and icon, witch one impact the most the performance?_
 |  N    |  Y    |  N   |   N   |
```
Comment (Tuizi) [inline dailyperf/tests/icon_border/test.md:13]: R: No impact on responsive time
A: Minor Impact difference if you render element during an animation
I: N/A
L: No impact on the load time (we exclude the impact of loading the webfont)

**Pull request merged**